internal/server/download.go

```diff
@@ -0,0 +1,50 @@
+package server
+
+import (
+	"errors"
+	"github.com/opg-sirius-finance-admin/internal/api"
+	"github.com/opg-sirius-finance-admin/internal/model"
+	"github.com/opg-sirius-finance-admin/internal/util/util"
+	"net/http"
+)
+
+type DownloadHandler struct {
```

Comment:
I see the other ones are named `{Method}{Action}Handler` so this should be renamed to `GetDownloadHandler` to match

Reply:
done